Tests/Integration/Carthage/project.yml

```diff
@@ -0,0 +1,13 @@
+name: CarthageTest
+options:
+  bundleIdPrefix: com.mapbox.common.events.carthage
+targets:
+  CarthageTest:
+    type: application
+    platform: iOS
+    deploymentTarget: "9.0"
+    sources: [Sources]
+    dependencies:
+      - framework: Carthage/Build/iOS/MapboxMobileEvents.framework
+    settings:
+        DEVELOPMENT_TEAM: "GJZR2MEM28"
```

Comment:
Is that `DEVELOPMENT_TEAM` correct?

Reply:
It's not being used for building and testing on a simulator, but it is the correct team id.